src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs

```diff
@@ -99,9 +99,10 @@ public Task<RestApiEndpoint> GetCheckGroupExistsEndpointAsync(string appName, st
 
         private async Task<RestApiEndpoint> GenerateRestApiEndpointAsync(string appName, string hubName, string pathAfterHub, TimeSpan? lifetime = null, IDictionary<string, StringValues> queries = null)
         {
-            var requestPrefixWithHub = $"{_serverEndpoint}api/{Version}/hubs/{Uri.EscapeDataString(GetPrefixedHubName(appName, hubName))}";
+            pathAfterHub = $"{pathAfterHub}?api-version={Version}";
```

Comment:
should we depend on `UriBuilder`.

Reply:
After reading docs of `UriBuilder`, I think the current way to generate API endpoint is more convinent